### ISSUE TITLE
Confusing error when `fuzz` subdir exists`

### ISSUE THREAD

**Issue by novafacing** [rust-fuzz/cargo-fuzz#341]
If you run `cargo fuzz init` in a crate where `fuzz` already exists, you get:

```
Error: could not find a cargo project
```

There should probably be an error for this specifically.

**Comment by fitzgen** [comment]
Agreed that we should have a better error/UX here. Would be very receptive to PRs that implemented this.

**Comment by novafacing** [comment]
Sure, happy to implement 🙂 consider me assigned.

**Comment by novafacing** [comment]
It looks like I might have had an old version, because now I see:

```
Error: failed to create directory /tmp/testlib/fuzz

Caused by:
    File exists (os error 17)
```

This seems fine, so maybe we can close this :)

**Comment by fitzgen** [comment]
Indeed. Thanks for filing an issue anyways!
